external/tomee/playlist.xml

```diff
@@ -27,7 +27,7 @@
 		</groups>
 	</test>
 	<test>
-		<testCaseName>tomee_test_j9</testCaseName>
+		<testCaseName>tomee_test</testCaseName>
```

Comment:
tomee is a microprofile tck test, so we should really name the directory and target as we do the others tomee_mp_tck.  In this case, since there are multiple targets in the playlist, this one would then get changed to tomee_mp_tck_openj9 and the target below it would be tomee_mp_tck_hotspot.